### ISSUE TITLE
custom entry types bug

### ISSUE THREAD

**Issue by mpgvalli** [JabRef/jabref#772]
If I import preferences one after another, the custom entry types add themselves. If I restart JabRef displays the custom entry types correct.
![unbenannt](https://cloud.githubusercontent.com/assets/17045510/12813389/0020a522-cb38-11e5-9142-dab252dd9ebd.PNG)

**Comment by mpgvalli** [comment]
And I can't delete the entry type "standard". If I set it to default and save the settings it returns and is also mentioned in the preferences files.

**Comment by jkomelj** [comment]
I have a similar problem – how to clear custom entry types which are identical to default ones. So, before entry type customization I have exported my preferences. After mistakenly setting some of entry types to default I have imported previously exported preferences. After restarting JabRef default settings are still valid although exported preferences in xml file were correct, e.g. 
            <entry key="customTypeName_0" value="Article"/>
            <entry key="customTypeReq_0" value="author;journal;number;pages;title;volume;year"/>
            <entry key="customTypeOpt_0" value="note;key"/>
but were not restored.

I have used the last snapshot JabRef_windows_3_3dev--snapshot--2016-03-09--master--f7a7ded
Best regards,  Janez Komelj
